torchdata/dataloader2/communication/iter.py

```diff
@@ -34,7 +34,17 @@ class NotAvailable(Exception):
 class InvalidStateResetRequired(Exception):
     """
     Returned by DataPipe when it is expecting to get reset request,
-    for example RouterDataPipe expecting all workers to request reset'
+    for example RouterDataPipe expecting all workers to request reset.
+    """
+
+    pass
```

Comment:
BTW, It might be better to add an Error message in `__init__`.
`super().__init__(msg)`.